src/main/java/software/amazon/cloudwatchlogs/emf/model/RootNode.java

```diff
@@ -45,7 +46,7 @@ class RootNode {
 
     RootNode() {
         aws = new Metadata();
-        properties = new HashMap<>();
+        properties = new ConcurrentHashMap<>();
```

Comment:
do we have UT cover this?

Reply:
I think so. This change is tested by invoking putProperty() in async context, which is covered by the test cases in MetricDirectiveThreadSafetyTest.java such as `testConcurrentPutProperty()` & `testConcurrentFlushAndMethodsOtherThanPutMetric()`.